### PR TITLE
Docker compose file for ARM processors

### PR DESCRIPTION
**Pull request by sumesh-aot**
## Changes
- Added new image for keycloak
- Added newly built image for bpm as openjdk image wasn't working with ARM

### CODE REVIEW

File: docker-compose/docker-compose.yml [hunk 1]
```diff
@@ -138,7 +138,7 @@ services:
     links:
       - forms-flow-webapi-db
       - forms-flow-forms
-    entrypoint: "/wait-for-service.sh forms-flow-webapi-db:5432 -s -- ./entrypoint.sh"  
+    entrypoint: "/wait-for-service.sh forms-flow-webapi-db:5432 -s -- /wait-for-service.sh keycloak:8080 -s -- ./entrypoint.sh "
```
Comment (sreehari-aot) [inline docker-compose/docker-compose.yml:141]: @sumesh-aot can you add forms-flow-forms to  the health check that was our major issue

**Review by sreehari-aot — APPROVED** [review]
LGTM!

**Review by alanraju-aot — APPROVED** [review]

File: docker-compose/docker-compose.yml (hunk 1 above)
Comment (sumesh-aot) [reply docker-compose/docker-compose.yml]: Added

**Review by sreehari-aot — APPROVED** [review]

**Review by abilpraju-aot — APPROVED** [review]
LGTM

**Pull request merged**